src/main/java/Duke.java

```diff
@@ -1,10 +1,63 @@
+import java.util.*;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    public static void main(String[] args) throws DukeException {
+        System.out.println("Leo: Yoooz it's your boy Leo! What's up?");
+        Scanner sc = new Scanner(System.in);
+        ArrayList<Task> tasks = new ArrayList<>();
+        while (true) {
+            System.out.print("You: ");
+            String str = sc.nextLine();
+            if (str.equals("bye")) {
+                System.out.println("Leo: Good talk man, catch you again some other time!");
+                break;
+            } else if (str.equals("list")) {
+                System.out.println("Leo: Here you go!");
+                for (int i = 0; i < tasks.size(); i++) {
+                    System.out.println(i + 1 + ". " + tasks.get(i));
+                }
+            } else if (str.split(" ")[0].equals("mark")) {
```

Comment:
I think for this "mark" block, you can do a try/catch block to catch IndexOutOfBoundException in case the user input is just "mark" instead of "mark 1". Applicable to other blocks that require a subsequent input after the keywords like unmark, delete, etc. as well :D